app/src/test/java/org/mozilla/fenix/settings/creditcards/DefaultCreditCardEditorControllerTest.kt

```diff
@@ -74,6 +90,7 @@ class DefaultCreditCardEditorControllerTest {
         coVerify {
             storage.deleteCreditCard(creditCardId)
             navController.popBackStack()
+            metrics.track(Event.CreditCardDelete)
```

Comment:
No good way to test the `Event.CreditCardAutofill` case?

Reply:
No, we currently don't have any testing on the Fenix end for the autofill services. We are going to extend this telemetry to include things like "saving a new card from autofill", which would involve some work on the AC side and will be testable. We might move this `CreditCardAutofill` case to a `Fact` at that point.

Reply:
We don't use the `CreditCardAutofill` service. I think you will need to go into AC and add a fact since this is technically handle by our `PromptFeature` and GV. We should be missing telemetry for 
credit_cards.manual_save which is probably the most important probe.